src/libraries/System.Linq/src/System/Linq/Range.SpeedOpt.cs

```diff
@@ -18,7 +18,7 @@ public int[] ToArray()
             {
                 int[] array = new int[_end - _start];
```

Comment:
```suggestion
                int[] array = new int[(nint)(uint)(_end - _start)];
```

This eliminates a sign-extension.

Reply:
This isn't worth the complexity.